src/deluge/hid/led/pad_leds.cpp

```diff
@@ -631,7 +631,7 @@ void renderExplodeAnimation(int32_t explodedness, bool shouldSendOut) {
 }
 
 void reassessGreyout(bool doInstantly) {
-	auto [newCols, newRows] = getUIGreyoutRowsAndCols();
+	auto [newCols, newRows] = getUIGreyoutColsAndRows();
```

Comment:
aren't these still swapped? I don't see a fix for this

Reply:
> aren't these still swapped? I don't see a fix for this

See ui.cpp line 70 for the real fix. all the rest are just name changes.

	return std::make_pair(cols, rows);